build-kernels-clang.sh

```diff
@@ -9,6 +9,7 @@ export CLANG_CC=$ANDROID_ROOT/prebuilts/clang/host/linux-x86/clang-r353983c/bin/
 
 # Mkdtimg tool
 export MKDTIMG=$ANDROID_ROOT/out/host/linux-x86/bin/mkdtimg
+export MKDTIMG=$ANDROID_ROOT/prebuilts/misc/linux-x86/libufdt/mkdtimg
```

Comment:
Duplicate, delete the previous line

Reply:
Yeah I forgot to add a comment, `MKDTIMG` and `CP_BLOB` can be dropped since these are now defined in the generic script.